common/display/displayplanemanager.cpp

```diff
@@ -213,10 +213,11 @@ bool DisplayPlaneManager::ValidateLayers(
 #endif
           plane->SetInUse(true);
           DisplayPlaneState &last_plane = composition.back();
+#ifndef DISABLE_VA
           if (layer->IsVideoLayer()) {
```

Comment:
Instead of this here, us it internal to SetVideoPlane. (i.e. it ignores setting the state).

Reply:
I think in this case while VA is disabled treat it as normal layer and not setting it as video plane is making more sense (not to call SetVideoPlane)?